src/EventProcessor/EventProcessorInterface.js

```diff
@@ -0,0 +1,39 @@
+goog.require('measurementlibrary.Storage.StorageInterface');
+goog.module('measurementlibrary.EventProcessor.EventProcessorInterface');
+
+/**
+ * A class implementing this interface can react to events pushed to the
+ * data layer.
+ * Event processors may perform actions such as sending a network request
+ * based on the event data, interacting with the storage interface to commit
+ * and retrieve values from long-term storage, or reading values from
+ * the abstract model.
+ *
+ * @interface
+ */
+class EventProcessor {
+  /**
+   * Process an event passed into the data layer.
```

Comment:
Nit: This is the wrong tense for method description. Applies to rest of PR as well.

Reply:
Good catch, fixed